### DISCUSSION TITLE
Threats Manager Support - 2.1.1.0 - Support for Windows Server OS ?

### DISCUSSION THREAD

**Discussion by rsrinivasanhome** [simonec73/threatsmanager#62] [General]
Threats manager studio 1.5.2   Supported Windows server OS .  Now with 2.1.1.0 version I am getting a message

![image](https://github.com/simonec73/threatsmanager/assets/31912418/c23d25cc-c71d-48f0-826c-5dd21a475edd)

for the following OS - Windows Server 2022 and Window Server 2019 data center

Even if Install .net framework 4.8.1 and restart the machine the error message does not go away

**Comment by simonec73** [comment]
I see that TMS itself raises the error. I assume you successfully installed it. I don't know what the cause is. I am not doing any specific checks in TMS. I can't promise I'll be able to fix this issue anytime soon. As you can see on the downloads page of the TMS site, I only support TMS on Windows 10 and 11 because I do not have a test environment for other OSs.

In any case, please consider that .NET Framework 4.8.1 supports Windows Server 2022 but not Windows Server 2019 (see https://learn.microsoft.com/en-us/dotnet/framework/get-started/system-requirements#supported-server-operating-systems). Another element to consider is that TMS is a 32 bits application. Therefore, it requires the 32-bit version of .NET Framework 4.8.1.
